gapis/api/vulkan/api/renderpass_framebuffer.api

```diff
@@ -589,6 +589,11 @@ sub ref!SubpassBeginInfo createSubpassBeginInfoFromInfo(const VkSubpassBeginInfo
   )
 }
 
+sub ref!SubpassEndInfo createSubpassEndInfo(const VkSubpassEndInfo* pSubpassEnd) {
+  _ = pSubpassEnd // this struct is empty and we do not support any extension for this struct
```

Comment:
This is done so the read/write observer callbacks are triggered for pSubpassEnd?

Reply:
No, it's more for the consistency. As soon as we get a new extension that uses it we need to change the function signature. Rather than that, it's better to pass the argument and not use it.

Reply:
sounds good